### ISSUE TITLE
Creating kinetic-devel branch?

### ISSUE THREAD

**Issue by wkentaro** [ros/nodelet_core#52]
@mikaelarguedas You stopped releasing new features to indigo and jade, so it is the time to create the kinetic-devel branch, right?

**Comment by mikaelarguedas** [comment]
@wkentaro Can you be more specific about the features you are referring to ?

1.9.8 has been released for all distros (c.f [status page comparison](http://repositories.ros.org/status_page/compare_indigo_jade_kinetic.html?q=nodelet))

I'm sometimes delaying releases on indigo and jade to allow more time for testing to be sure not to break anything on the previous distros but the goal for now is to avoid branching out as long as there are not big changes preventing to do so.

**Comment by wkentaro** [comment]
I'd like to use this: https://github.com/ros/nodelet_core/commit/e5cb1982fbb73e8b2887c2b5740d0d11fd5cf78a

But seeing [ros/rosdistro](https://github.com/ros/rosdistro/pulls?q=is%3Apr+nodelet_core+is%3Aclosed), it does not seem released because PR for 1.9.6-0 was closed:
https://github.com/ros/rosdistro/pull/13041

**Comment by wkentaro** [comment]
@mikaelarguedas 
Oops, sorry, maybe I'm missing something.

I'm using shadow-fixed for apt and installed nodelet via apt,
and found the nodelet_lazy.h file as shown in below:

```
% source /opt/ros/indigo/setup.zsh

% rosversion nodelet
1.9.8

% ls /opt/ros/indigo/include/nodelet_topic_tools/nodelet_lazy.h
/opt/ros/indigo/include/nodelet_topic_tools/nodelet_lazy.h
```

**Comment by mikaelarguedas** [comment]
1.9.6 has not been release because the timing between the indigo sync and the testing of these changes was not optimal. I closed it in favor of 1.9.7 that includes both the changes of #45 and #49. So you should be able to leverage both.
Thanks for both these contributions BTW :+1:

**Comment by wkentaro** [comment]
Thank you!
